### ISSUE TITLE
Add kinship relationships to RO

### ISSUE THREAD

**Issue by cmungall** [oborel/obo-relations#648]
@balhoff @mellybelly and I are working with the GA4GH KIN group. They have created an ontology of kinship relations:

https://github.com/GA4GH-Pedigree-Standard/family_history_terminology/

I identified a few issues:

https://github.com/GA4GH-Pedigree-Standard/family_history_terminology/issues

There is a positive reception to the idea of housing these in RO. There would need to be some extract function that would extract the kin subset.

One challenge is that RO is species-neutral and focused on biological relationships. KIN is human-centric (perhaps also used for veterinary animals?) and includes a mix of social and biological relationships, e.g. step-sibling of, adopted parent of, as well as encompassing situations like surrogacy.

Possible approaches:

1. House biological relations in RO and map to KIN
2. House all of KIN in RO, and provide method to extract subset relevant for KIN use case

**Comment by cmungall** [comment]
update from @mellybelly - the GA4GH community would like a namespace they control

**Comment by wdduncan** [comment]
@cmungall I assume you mean that they don't want the IRIs to have the form `RO:xxxxxxx`. Right?
Does mean we should close this issue?

**Comment by cmungall** [comment]
No, we still need biological kinship relationships in RO as these are broadly useful. I propose we follow route 1 I outlined above (House biological relations in RO and map to KIN)

**Comment by addiehl** [comment]
Werner Clusters' [recent paper on kinship](http://www.referent-tracking.com/RTU/files/Rabenberg-Pengput-Ceusters-ICBO2023-for-publication.pdf)

**Comment by cmungall** [comment]
Update on this:

- There are still a number of logical issues on the GA4GH kinship ontology (https://github.com/GA4GH-Pedigree-Standard/family_history_terminology/issues/created_by/cmungall) 
- we still haven't made any progress on this in RO

**Comment by sabrinatoro** [comment]
We need some kinship relations that are non-human specific for VBO (Vertebrate Breed Ontology).

**Comment by balhoff** [comment]
Recent paper to keep in mind: http://www.referent-tracking.com/RTU/files/Rabenberg-Pengput-Ceusters-ICBO2023-for-publication.pdf
